src/base/memory_hash.c

```diff
@@ -1323,6 +1536,44 @@ mht_get2 (const MHT_TABLE * ht, const void *key, void **last)
   return NULL;
 }
 
+/*
+ * mht_get_hls - Find the next data associated with the key; Search the entry next
+ *            to the last result
+ *   return: the data associated with the key, or NULL if not found
+ *   ht(in):
+ *   key(in):
+ *   last(in/out):
+ *
+ * NOTE: This call does not affect the LRU list.
+ */
+void *
+mht_get_hls (const MHT_HLS_TABLE * ht, const void *key, void **last)
+{
+  unsigned int hash;
+  HENTRY_HLS_PTR hentry;
+
+  assert (ht != NULL && key != NULL);
```

Comment:
is not necessary to check the **last** variable ?

Reply:
okay. i'll fix it